src/webhandler.ts

```diff
@@ -82,6 +82,10 @@ export async function githubRawFixtureRequest(path: string): Promise<object | vo
   return request(url);
 }
 
+export async function githubRawManufacturersRequest(): Promise<object | void> {
```

Comment:
```suggestion
export async function githubRawManufacturersRequest(): Promise<Manufacturers | void> {
```

No reason not to yeah? `object` is generally not what you want. our other project has a linting rule against it.